loader/loader.go

```diff
@@ -979,6 +979,7 @@ func (l *Loader) restoreData(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	defer l.toDB.CloseBaseConn(baseConn)
```

Comment:
`l.toDB.CloseBaseConn(baseConn)` for every start/resume operation, but only close `toDBConns` when closing the whole load unit?

when an error occurred for any connection in `toDBConns`, no reset/re-apply for the connection, so it can't work anymore unless _stop_/_start_ again?

Is it possible to document the DB/Connection usage, ownership and life cycle in some places both for load and sync?

Reply:
I unify the Loader/Syncer reset connection,  for every start/resume operation, the connection of both Loader/Syncer will be reset and I add a simple document in [here](https://github.com/pingcap/dm/pull/266/files#diff-e25c1def89789065c4904c1d5adc0867R29-R54), please check it